javascript/sdk/lib/types.ts

```diff
@@ -399,3 +425,18 @@ export type PartnerEndpoints = Record<
     state: string;
   }
 >;
+
+export interface AuditLog {
+  name: string;
+  account_id: string;
+  user_id: string;
+  values: AuditLogValues;
+  status: ExitCode;
+  timestamp: string;
+}
+
+export interface AuditLogValues {
```

Comment:
Is there the idea of "any" or "blob" in js? `values` holds different fields for different events and ideally is a blackbox dictionary (ex. create_user will probably have permission and handle, upload_attachment will have test_id and filename, etc)

Reply:
@kenrick So im thinking we could make the "values" in AuditLog the type of "any" or perhaps just an object?

Reply:
type AuditLogValues = Record<string, unknown>

Reply:
ah that would make more sense